Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -517,10 +536,10 @@ public void indexConcepts(String userName)
 
         }
 
+        
         PerFieldAnalyzerWrapper perFieldAnalyzerWrapper = new PerFieldAnalyzerWrapper(standardAnalyzer,
-                analyzerPerField);
-
-        QueryBuilder qBuild = new QueryBuilder(perFieldAnalyzerWrapper);
+                    analyzerPerField);
+        QueryBuilder qBuild = new QueryBuilder(customAnalyzer);
```

Comment:
this should still have the perfield analyzer

Reply:
removed customAnalyzer and added the perfieldanalyzer

Reply:
Yup, Got it. replaced the standardAnalyzer.